src/Platform/Dashboard.php

```diff
@@ -384,8 +384,8 @@ public function registerMenuElement(string $location, \Orchid\Screen\Actions\Men
     public function renderMenu(string $location): string
     {
         return $this->menu->get($location)
-            ->sort(function (Menu $menu) {
-                return $menu->get('sort', 0);
+            ->sort(function (Menu $menuA, Menu $menuB) {
```

Comment:
Hello, let's change the name of the variables to something more meaningful, for example:

```php
function (Menu $current, Menu $next)
```

Reply:
Sorry, done.